tests/providers/cloudera/model/test_connection.py

```diff
@@ -0,0 +1,87 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Tests related to the CDE connection"""
+
+from __future__ import annotations
+
+import json
+import unittest
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from airflow.providers.cloudera.model.connection import CdeConnection
+
+
+class CdeConnectionTest(unittest.TestCase):
+
```

Comment:
We try to use native `pytest`, there are exists some old tests which use `unittest.TestCase` for historical reason 
(sooner or later we would replace all of them)

Some suggestion

- Get rid of `unittests.TestCase` class and **TestCase.assert*** methods
- Convert classes **setUp*** and **tearDown*** to [appropriate pytest methods](https://docs.pytest.org/en/6.2.x/xunit_setup.html#classic-xunit-style-setup)

Reply:
Thanks you, will take some time to shift existed tests.